android/app/src/main/java/com/gpslogger/LocationModule.java

```diff
@@ -66,6 +66,7 @@ public String createLocationJSON(Location location) throws JSONException {
         JSONObject json = new JSONObject();
         json.put("longitude", location.getLongitude());
         json.put("latitude", location.getLatitude());
+	json.put("altitude", location.getAltitude());
```

Comment:
I think we should use spaces instead of tabs.
I'm currently using 4 space indentation.